src/server/scripts/EasternKingdoms/Karazhan/boss_netherspite.cpp

```diff
@@ -236,8 +236,9 @@ struct boss_netherspite : public BossAI
         scheduler.Schedule(30s, [this](TaskContext)
         {
             SwitchToPortalPhase();
+            me->GetThreatMgr().ResetAllThreat();
```

Comment:
This was done in 76d1d02dce2f63348dccb8ac218ec0cc54e6ce8c with `DoResetThreatList()`. Would that not work here as well or is there a reason it's not preferred?

Reply:
Correct, Bench beat me to it. This is the safe method to use